osu.Game/Screens/Edit/Verify/IssueList.cs

```diff
@@ -43,7 +46,7 @@ private void load(OverlayColourProvider colours)
             generalVerifier = new BeatmapVerifier();
             rulesetVerifier = beatmap.BeatmapInfo.Ruleset.CreateInstance().CreateBeatmapVerifier();
 
-            context = new BeatmapVerifierContext(beatmap, workingBeatmap.Value, verify.InterpretedDifficulty.Value);
+            context = new BeatmapVerifierContext(beatmap, workingBeatmap.Value, verify.InterpretedDifficulty.Value, beatmapInfo => beatmapManager.GetWorkingBeatmap(beatmapInfo));
```

Comment:
I really have to wonder whether this isn't stopping a step short by stopping at `.GetWorkingBeatmap()`. Something tells me that sooner rather than later you're going to want to call `.GetPlayableBeatmap()` on the result of this as well, if you want to get beatmaps which are in a *usable and inspectable* state going forward.